pkg/profiles/openapi.go

```diff
@@ -109,7 +106,8 @@ func swaggerToServiceProfile(swagger spec.Swagger, namespace, name, clusterDomai
 	return profile
 }
 
-func mkRouteSpec(path, pathRegex string, method string, operation *spec.Operation) *sp.RouteSpec {
+// MkRouteSpec makes a service profile route from an OpenAPI operation.
+func MkRouteSpec(path, pathRegex string, method string, operation *spec.Operation) *sp.RouteSpec {
```

Comment:
I'll probably leave this in here. It's used by `swaggerToServiceProfile` in this file quite a few times, so I think making it public in `pkg` package is okay.

Reply:
ah, my thinking was this isn't JUST used by the openapi code, it makes sense to go in `pkg/profiles/profiles.go`.  It's a bit weird for non-openapi code to depend on a function defined in `openapi.go` (not that REALLY matters since it's in the `pkg/profiles` package, it just feels weird)